### PR TITLE
(Ledger Store Benchmark) Display storage size of all data shreds

### PR DESCRIPTION
**Pull request by yhchiang-sol**
#### Summary of Changes
Display the storage size of all data shreds in the ledger store benchmark.

This allows the benchmark to track whether the compactions are triggered at the right time.

This PR is based on top of #22443 (APIs for obtaining physical size of all data and coding shreds)

#### Test Plan / Results
Run the ledger cleanup benchmark and observe the shred storage size and its delta are correctly reported.
(The last two numbers before the three 0.00)

```
[2022-01-12T03:09:01.890862000Z INFO  ledger_cleanup::tests] Bench info num_batches: 500000, batch size (slots): 2, shreds_per_slot: 5, num_shreds_total: 5000000
TIME_MS,DELTA_MS,START_SLOT,BATCH_SIZE,SHREDS,MAX,SIZE,DELTA_SIZE,DATA_SHRED_SIZE,DATA_SHRED_SIZE_DELTA,CPU_USER,CPU_SYSTEM,CPU_IDLE
[2022-01-12T03:09:01.890938000Z INFO  ledger_cleanup::tests] 0,0,0,0,0,0,404598,404598,0,0,0.00,0.00,0.00
[2022-01-12T03:09:01.890943000Z INFO  ledger_cleanup::tests] Begin inserting shreds ...
[2022-01-12T03:09:01.891170000Z INFO  ledger_cleanup::tests] 0,0,1,2,5,50,404598,0,0,0,0.00,0.00,0.00
[2022-01-12T03:09:02.396330000Z INFO  ledger_cleanup::tests] 505,505,30049,2,5,50,183170142,182765544,0,0,0.00,0.00,0.00
[2022-01-12T03:09:02.901472000Z INFO  ledger_cleanup::tests] 1010,505,58883,2,5,50,615911082,432740940,257467805,257467805,0.00,0.00,0.00
[2022-01-12T03:09:03.406592000Z INFO  ledger_cleanup::tests] 1515,505,86843,2,5,50,785865258,169954176,257467805,0,0.00,0.00,0.00
[2022-01-12T03:09:03.911731000Z INFO  ledger_cleanup::tests] 2020,505,115631,2,5,50,1218091867,432226609,514935956,257468151,0.00,0.00,0.00
[2022-01-12T03:09:04.416920000Z INFO  ledger_cleanup::tests] 2525,505,144013,2,5,50,1648143847,430051980,772403758,257467802,0.00,0.00,0.00
...
```

### CODE REVIEW

**Review by steviez — APPROVED** [review]
LGTM!

Outside of the scope of this PR, but I might look to see if we can clean this print up a little. I find it hard to follow since the header is only printed once, and the datapoints might be mixed in with other log statements

**Comment by yhchiang-sol** [comment]
> Outside of the scope of this PR, but I might look to see if we can clean this print up a little. I find it hard to follow since the header is only printed once, and the datapoints might be mixed in with other log statements

Yep, agreed.  The output / debug messages of this benchmark is a bit messy as they don't follow the same format.
